### ISSUE TITLE
Constructor parameters showing up as body param in docs

### ISSUE THREAD

**Issue by chriskessel** [swagger-api/swagger-core#1461]
I'm using Dropwizard 0.8.2 and swagger-jersey2-jaxrs 1.5.3-M1.

I can't figure out how to disable this extra body param in the generated swagger.json:

    @Api(value = "/foo/{outputFormat}", description = "foo")
    @Path("/foo/{outputFormat}")
    @Produces({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON})
    public class RoutesResourceRest {
       private final MyDelegate delegate;

        @Inject
        public RoutesResourceRest(MyDelegate delegate) {
            this.delegate = delegate;
        }

        @GET
        @ApiOperation(value = "blah",
                response = MyResponse.class,
                notes = "Returns a collection of things.")
        @ApiResponses({...})
        @Path("subpath/goes/here")
        public Response doFoo(@PathParam("outputFormat") String outputFormat) {
            ...
        }
    }

Even though it's a GET, the swagger.json generated for the endpoint has a body parameter. How do I disable that?
```javascript
"paths": {
"/foo/{outputFormat}/subpath/goes/here": {
"get": {
"tags": [
"foo{outputFormat}"
],
...
"parameters": [
{
"in": "body",
"name": "body",
"required": false,
"schema": {
"$ref": "#/definitions/MyDelegate"
},
...
}
'''

**Comment by chriskessel** [comment]
Found it. io.swagger.jaxrs.utils.ReflectionUtils.java:158

            if ("javax.inject.Inject".equals(annotation.annotationType().getName())) {

In a previous project I used Google's inject and didn't get this mysterious body parameter. In my project showing the issue, I'm using javax.inject.Inject.  Guice doesn't care which inject I use, but I try to use standard Java when possible so that's why I switched to javax.inject.Inject. 

I can switch back to Google's inject, but I guess my original question is still valid: how do I prevent this @Inject annotation from creating the body param in the swagger.json?
